package.json

```diff
@@ -44,9 +44,9 @@
         "@types/shelljs": "^0.8.0",
         "inquirer": "^6.0.0",
         "shelljs": "^0.8.2",
-        "ts-node": "^7.0.0",
```

Comment:
Is this necessary for this change?

Reply:
It was just for an `npm audit fix` that's all.